### DISCUSSION TITLE
Browser console errors after upgrading from 5.x to 6.x

### DISCUSSION THREAD

**Discussion by ChimpGamer** [Power-Components/livewire-powergrid#1743] [BUG]
After upgrading **ONLY** powergrid from 5.x to 6.x I keep getting errors spamming in the browser console.
These are the errors that keep popping up in the browser console:
[127.0.0.1-1729352399439.log](https://github.com/user-attachments/files/17445835/127.0.0.1-1729352399439.log)

**Comment by luanfreitasdev** [comment]
The only way to get these errors is if you don't configure your project correctly. Make sure that powergrid.js is imported into app.js

```js
 import './../../vendor/power-components/livewire-powergrid/dist/powergrid'
```

![image](https://github.com/user-attachments/assets/e6e39800-0642-4482-9e5b-90ec2c0e857a)

**Reply by ChimpGamer** [reply]
My app.js file imports the powergrid script.
![image](https://github.com/user-attachments/assets/9876cc31-524a-49c8-95d0-94b444c4539b)

**Reply by ChimpGamer** [reply]
Hey, I'm sorry. I just managed to figure out the issue. For some reason the app.js wasn't being being loaded by vite.
